src/Views/IframeView.php

```diff
@@ -94,32 +295,20 @@ public function setBodyClasses( $classes ) {
 	 */
 	public function render() {
 		ob_start();
```

Comment:
Does this need to use output buffering?

Reply:
@bradp I do not want to break accidentally. that why I am storing HTML output first in the buffer then returning this.

Reply:
👍